### ISSUE TITLE
Add GA

### ISSUE THREAD

**Issue by lambtron** [denoland/denobyexample#20]
Adding server side GA so we can measure traffic on the examples page.

UA-property should match the same as deno.land. Shouldn't be an issue due to the same TLD.

**Comment by lino-levan** [comment]
Would love to get some traffic numbers on denobyexample but I don't think I'm able to push this forward without deeper access to the project.

We've gotten some very positive comments on discord:

> yeah the example page is much more helpful

> Oh wow. I learned most by trial and error. Wish i knew about this.

> I've been inspired by the documentation [at denobyexample]

> [This is] amazing! Thanks. I will try this immediately

One common message I'm seeing is something along the lines of "I wish I knew about this sooner" so maybe we should try to link this in a more obvious manner from other deno resources (the manual has a few hard to find links).

Thoughts @hashrock?

**Comment by hashrock** [comment]
Yeah it's nice to have. Deno by Example is not very well known and needs to be improved.

However, in order to add GA, environment variables need to be added to Deploy, and Deno Deploy has a restriction that does not allow to generate preview deployment on external PR when using environment variables. This is to prevent environment variable theft.

I think it's inconvenient to not be able to use Preview when there are many PRs from outside, like in this repository.
We may want to wait to introduce it until we really need it.
